demo/package.json

```diff
@@ -1,14 +1,15 @@
 {
   "nativescript": {
-    "id": "org.nativescript.demotwilio",
-    "tns-ios": {
-      "version": "5.2.0"
-    },
+    "id": "com.hiveku.twilioplugindev",
```

Comment:
Could you revert the `id `this change? Also, I would like to have the latest 5.2.0 version instead of the 5.1.1 one, if there is no good reason for it.

Reply:
Switched to 5.2.0 and restored the old package ID 💯